conf.d/spark.yaml.example

```diff
@@ -0,0 +1,20 @@
+init_config:
+
+instances:
+  #
+  # The Spark check retrieves metrics from YARN's ResourceManager. This
+  # check must be run from the Master Node and the ResourceManager URI must
+  # be specified below. The ResourceManager URI is composed of the 
+  # ResourceManager's hostname and port.
+  #
+  # The ResourceManager hostname can be found in the yarn-site.xml conf file
+  # under the property yarn.resourcemanager.address
+  #
+  # The ResourceManager port can be found in the yarn-site.xml conf file under
+  # the property yarn.resourcemanager.webapp.address
+  #
+  - resourcemanager_uri: http://localhost:8088
+    cluster_id: MySparkCluster
```

Comment:
Ok, makes sense.

So let's keep the `cluster_id` provided by Yarn. That'll also ensure that metrics from different clusters have different sets of tags (and don't overwrite one another), so the `cluster_name` can be optional and defined by the user as one of the `tags` in the instance config.

We could recommend the user to set a `cluster_name` tag in the yaml by setting the first optional tag to `cluster_name:MySparkCluster` for instance, but keep it commented out.